reinforcement_learning/testnotebooks.txt

```diff
@@ -0,0 +1,2 @@
+rl_cartpole_ray/rl_cartpole_ray_gymEnv.ipynb
+rl_cartpole_coach/rl_cartpole_coach_gymEnv.ipynb
```

Comment:
Minor: Can you add comment somewhere in the test config file that only 1 instance_count test case is applied to the coach notebook to avoid possible confusion. Just for future maintenance.

Reply:
Sure, I'll add that.